libs/features/mas/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
what about adding **/ prefix to root .gitignore?

Reply:
Fixed